src/pages/graphql/schema/checkout/mutations/handle-payflow-pro-response.md

```diff
@@ -63,24 +67,3 @@ mutation {
   }
 }
 ```
-
-## Input attributes
-
-The `PayflowProResponseInput` object must contain the `cart_id` and `paypal_payload` attributes.
-
-Attribute |  Data Type | Description
---- | --- | ---
-`cart_id` | String! | The unique ID that identifies the customer's cart
-`paypal_payload` | String! | The payload returned from PayPal
-
-## Output attributes
-
-The PayflowProResponseOutput contains a `Cart` object.
-
-### Cart object
-
-import CartObject from '/src/pages/_includes/graphql/cart-object-24.md'
-
-<CartObject />
```

Comment:
I reckon other files are using this include. Is the plan to do a sweep after you've finished linking to the reference for the entire API?

If so, will you please add a story to your epic about removing the exclude pattern from the `test:links` npm script when that's done?

Reply:
It's [COMDOX-558](https://jira.corp.adobe.com/browse/COMDOX-558)